core/src/com/nukesz/game/MenuScreen.java

```diff
@@ -0,0 +1,84 @@
+package com.nukesz.game;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
+import com.badlogic.gdx.utils.viewport.ScreenViewport;
+
+public class MenuScreen extends AbstractGameScreen {
+
+    private Stage stage;
+
+    public MenuScreen(SnakeGame game) {
+        super(game);
+        stage = new Stage(new ScreenViewport());
+        Gdx.input.setInputProcessor(stage);
+    }
+
+    @Override
+    public void show() {
+        Table table = new Table();
+        table.setFillParent(true);
+        stage.addActor(table);
+
+        // temporary until we have asset manager in
+        Skin skin = new Skin(Gdx.files.internal("skin/glassy-ui.json"));
+
+        TextButton play = new TextButton("Play", skin);
+        TextButton options = new TextButton("Options", skin);
+        TextButton exit = new TextButton("Exit", skin);
+
+        table.add(play).fillX().uniformX();
+        table.row().pad(10, 0, 10, 0);
+        table.add(options).fillX().uniformX();
+        table.row();
+        table.add(exit).fillX().uniformX();
+
+        play.addListener(new ChangeListener() {
+            @Override
+            public void changed(ChangeEvent event, Actor actor) {
+                game.setScreen(new GameScreen(game));
+            }
+        });
```

Comment:
If possible, please rewrite this and the folliwing to addlistener to lambda expression.